sotopia/database/evaluation_dimensions.py

```diff
@@ -0,0 +1,232 @@
+from redis_om import JsonModel
+from redis_om.model.model import Field
+from pydantic import BaseModel, create_model
+from typing import Type, Callable, Tuple, Annotated, Union
+
+
+class CustomEvaluationDimension(JsonModel):
+    name: str = Field(index=True)
+    description: str = Field(index=True)
+    range_high: int = Field(index=True)
+    range_low: int = Field(index=True)
+
+
+class CustomEvaluationDimensionList(JsonModel):
```

Comment:
can we use this as a model to save a set of dimensions? like same name sotopia, then it automatically retrieve all the sotopia original dimensions and would be ready to use

Reply:
Yes that is what I am thinking. Do we want to allow different evaluation metrics to have the same name?

Reply:
For example, there is an original sotopia dimension and a refined one, say
[old] goal: provide a goal score of 1-10 where higher score indicates higher completion
[new] goal: provide a goal score of 1-10, where 1-3: xxx, 4-6: yyy

I think this is something we do not want to see, but sometimes we might need to have these two at the same time?